### PR TITLE
PP-8337: Add tasks to start and stop RDS instances

### PR DESCRIPTION
**Pull request by jfharden**
Added tasks to:

1. Start an rds instance and wait for it to reach the available state
2. Stop an rds instance and wait for it to reach the stopped state

# Examples of running:
## Start RDS instance

Where databases were started from a stopped state: https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/perf-tests/jobs/scale-up-databases/builds/9

Where databases were already starting: https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/perf-tests/jobs/scale-up-databases/builds/8

Where databases were already started: https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/perf-tests/jobs/scale-up-databases/builds/10

## Stop RDS instance

Where databases were stopped from an available state: https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/perf-tests/jobs/scale-down-databases/builds/8

Where some databases were already stopped and some were already stopping: https://pay-cd.deploy.payments.service.gov.uk/teams/pay-dev/pipelines/perf-tests/jobs/scale-down-databases/builds/7

### CODE REVIEW

File: ci/tasks/start-rds-instance.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          "It's currently in ${CURRENT_STATE}, manual intervention will be required to resolve this."
+
+        exit 1
+      else
+        echo "Starting RDS instance ${RDS_INSTANCE_NAME}"
+        aws rds start-db-instance --db-instance-identifier "${RDS_INSTANCE_NAME}" >> /dev/null
+      fi
+
+      echo "Waiting for RDS instance ${RDS_INSTANCE_NAME} to be available"
+      aws rds wait db-instance-available --db-instance-identifier "${RDS_INSTANCE_NAME}"
```
Comment (oswaldquek) [inline ci/tasks/start-rds-instance.yml:40]: Might it be useful to have another line after this to say "RDS instance is now available"?

**Comment by oswaldquek** [comment]
Just checking we don't need to pause the deploy-to-perf before scaling up databases?

File: ci/tasks/start-rds-instance.yml (hunk 1 above)
Comment (jfharden) [reply ci/tasks/start-rds-instance.yml:40]: Ah yeah good plan

**Comment by jfharden** [comment]
> Just checking we don't need to pause the deploy-to-perf before scaling up databases?

We do, but that will be part of the pipeline rather than in this task. When this task is run from inside the deploy-to-perf pipeline in the migration tasks it needs to keep the pipeline unpaused

File: ci/tasks/start-rds-instance.yml (hunk 1 above)
Comment (jfharden) [reply ci/tasks/start-rds-instance.yml:40]: Done :)

**Review by oswaldquek — APPROVED** [review]

**Pull request merged**